js/skin.js

```diff
@@ -300,4 +301,4 @@ Skin.defaultProps = {
 
 };
 
-module.exports = Skin;
\ No newline at end of file
+module.exports = Skin;
```

Comment:
This is done automatically by Atom, let me check to see if I can make it not do that.